reports/create_tables.Rmd

```diff
@@ -34,6 +34,12 @@ df_consultation_med_counts <- read_csv(
   col_types = cols(dmd_code = col_character())
 )
 
+vmp_lookup <- read_csv(
+  here("lib", "reference", "vmp_vtm_lookup.csv"),
+  col_types = cols(id = col_character())
+) %>%
+  rename(code = id)
```

Comment:
Can we add a bit more information about this lookup table as a comment above? Where does it come from, how was it created, how could we reproduce it? Does it need to be updated?